### DISCUSSION TITLE
Notion

### DISCUSSION THREAD

**Discussion by swooyuk** [catppuccin/catppuccin#2564] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://www.notion.so

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by stellophiliac** [comment]
i made a personal macchiato theme over a year ago, if that'd be useful at all. i'm not sure how compatible it is with notion now, though

https://gist.github.com/stellophiliac/d251d3f7db242a6bdbcae3cff0f2dc8c

based off [this version](https://github.com/OlaoluwaM/notion-catppuccin)

**Comment by sidheandtea** [comment]
**tl;dr** - I'm still trying to figure out the best way to contribute it, but I have a color library and color palette database setup in Notion that I'm adding catppuccin flavors to right now.

using [catppuccin/palette/palette.json@main](https://github.com/catppuccin/palette/blob/main/palette.json) ( [v1.8.0](https://github.com/catppuccin/palette/releases/tag/v1.8.0)) converted to `.xlsx` (all palettes table + tab per flavor) and `.csv`.

More details:

Notion itself doesn't let you change colors (at least at the free level) beyond dark and light, but the color palettes can be displayed as blocks, and you can use https://placehold.co to create a placeholder image filled with the color (example: [frappé : rosewater](https://placehold.co/1902x1080/f2d5cf/f2d5cf.png)).
<img width="300" height="100" alt="f2d5cf 1" src="https://github.com/user-attachments/assets/1c04bc30-d322-40a4-8fd9-86afc605ad82" />
My Notion databases only use the Hex codes, but the way that I created the Excel sheet can easily include the other columns if needed.
